govcd/lbvirtualserver_test.go

```diff
@@ -100,6 +99,11 @@ func (vcd *TestVCD) Test_LBVirtualServer(check *C) {
 	check.Assert(lbVirtualServerById.ID, Equals, lbVirtualServerByName.ID)
 	check.Assert(lbVirtualServerById.Name, Equals, lbVirtualServerByName.Name)
 
+	// GetLbVirtualServers should return at least one vs which is our.
+	servers, err := edge.GetLbVirtualServers()
+	check.Assert(err, IsNil)
+	check.Assert(servers, Not(HasLen), 0)
```

Comment:
We should check at least some of the server fields, to make sure they are as expected

Reply:
The problem is that if the testing environment is not clean (e.g. there is more than 1VS/Pool/etc.) we cannot be sure what are correct values. Generally we should not expect correct values, since we're not filtering by anything - we want to get everything without expecting something specific (for which there is get by id, get by name funcs)